card/mod/05_standard/set/all/rich_html/wrapper.rb

```diff
@@ -10,7 +10,11 @@ def slot_options args
     options_hash[:subslot] = 'true' if args[:subslot]
 
     @@slot_option_keys.inject(options_hash) do |hash, opt|
-      hash[opt] = args[opt] if args[opt].present?
+      if args[opt].present?
+        hash[opt] = args[opt]
+      elsif Env.params['slot'].present? && Env.params['slot'][opt.to_s].present?
+        hash[opt] = Env.params['slot'][opt.to_s]
+      end
```

Comment:
that's making a mess.   The question is why the args aren't already there as they should be.